src/main/java/ch/njol/skript/command/ScriptCommand.java

```diff
@@ -131,7 +131,7 @@ public class ScriptCommand implements TabExecutor {
 	 */
 	public ScriptCommand(
```

Comment:
To avoid a breaking change we could retain the old constructor as well and wrap the string in a dummy `CommandUsage`.